lua/nginx_lua_zipstream.lua

```diff
@@ -131,6 +126,20 @@ local stream_zip = function(file_list)
         local desc = {
             istext = true,
             isfile = true,
+            isdir = false,
+            platform = 'unix',
+            exattrib = {
+                -- unix permissions: "-rw-rw-rw-""
```

Comment:
It looks like it's actually `-rw-rw---`

Reply:
Yeah, I changed it later to remove world read/write. Probably not a good idea on a Linux system to have it world accessible.